src/averageAges.js

```diff
@@ -14,13 +51,17 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+
+function calculateMenAverageAge(people, century = 21) {
+  const men = getMen(people);
+  const menDiedInCentury = men
+    .filter(man => getCentury(man['died']) === century);
+
+  const menAges = getAges(menDiedInCentury).length !== 0
+    ? getAges(menDiedInCentury) : getAges(men);
+
+  return getAverage(menAges);
+};
```

Comment:
```suggestion
  const men = century === undefined
    ? getMen(people)
    : getMen(people)
      .filter(man => (
        getCentury(man['died']) === century
      ));

  const menAges = getAges(men);

  return getAverage(menAges);
```

Check if `century === undefined` on the first step of your algorithm

Reply:
done